dev/io.openliberty.checkpoint_fat_mpconfig/fat/src/io/openliberty/checkpoint/fat/MPConfigTest.java

```diff
@@ -120,18 +99,15 @@ private void configureBeforeRestore() {
                 case annoValueTest:
                     removeVariableConfig("req_scope_key");
                     break;
-                case envValueChangeTest:
-                    configureEnvVariable(server, singletonMap("req_scope_key", "envValueChange"));
+                case varDirValueTest:
+                    server.copyFileToLibertyServerRoot("variables", "configVariables/req_scope_key");
                     break;
                 case noDefaultEnvValueTest:
                     configureEnvVariable(server, singletonMap("optional_req_scope_key", "optionalEnvValue"));
                     break;
                 case noDefaultServerValueTest:
                     updateVariableConfig(server, "optional_req_scope_key", "optionalServerValue");
                     break;
-                case providerEnvValueChangeTest:
```

Comment:
`providerEnvValueTest` can come here.

Reply:
Nice catch.  Yes I need to add two provider tests: one for req scope and one for app scope